tests/console/copy_iso_to_external_drive.pm

```diff
@@ -0,0 +1,44 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Copy the installation ISO to an external drive
+# Maintainer: Joachim Rauch <jrauch@suse.com>
+# Tags: boo#1040749
+
+use base 'btrfs_test';
+use strict;
+use testapi;
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
+
+    # choose unpartioned disk and set <$disk> shell variable
+    $self->set_playground_disk;
+    my $DISK = get_var('PLAYGROUNDDISK');
+    #partition HDD2
+    assert_script_run "echo Disk: $DISK";
+    assert_script_run "parted $DISK mklabel gpt",                       240;
+    assert_script_run "parted -a opt $DISK mkpart primary ext4 0% 50%", 240;
+    my $PARTITION = $DISK . "1";
+    assert_script_run "mkfs.ext4 $PARTITION", 240;
+
+    #mount HDD2
+    assert_script_run "mount $PARTITION /mnt";
+
+    #copy iso from DVD to HDD2
+    assert_script_run 'dd if=/dev/dvd of=/mnt/install.iso', 3000;
+
+    #check if copy worked
+    assert_script_run '[ -s /mnt/install.iso  ]';
```

Comment:
Yes. `[ -s ]` only checks that the file is of non-zero size. E.g. `md5sum /dev/dvd` and `md5sum /mnt/install.iso` should do the trick for you. Finding suitable hash algo, up to you :).